src/ConfigResource.php

```diff
@@ -35,32 +35,33 @@ class ConfigResource
     /**
      * @var ConfigWriter
      */
-    protected $writer;
+    protected $configWriter;
```

Comment:
Why this property is renamed? It's `protected` and the class is not `final`, so it is 'minor BC break'.

Reply:
Thanks, that's a valid point.  I dislike abbreviations in code; call it what it is, not a nickname; but as you point out this isn't final that's a good catch.